cg/meta/upload/scout/balsamic_config_builder.py

```diff
@@ -36,10 +38,10 @@ def include_sample_files(self, config_sample: ScoutBalsamicIndividual) -> None:
 
         sample_id: str = config_sample.sample_id
         if config_sample.alignment_path:
-            if "tumor" in config_sample.alignment_path:
-                sample_id = "tumor"
-            elif "normal" in config_sample.alignment_path:
-                sample_id = "normal"
+            if SampleType.TUMOR.value in config_sample.alignment_path:
+                sample_id = SampleType.TUMOR.value
```

Comment:
For consistency keep it always either as "SampleType.TUMOR/NORMAL.value" or "SampleType.TUMOR/NORMAL", sometimes is specified in one way and sometimes in the other. I'd go for the second one.

Reply:
There is a reason behind it:
<img width="223" alt="Screenshot 2023-01-31 at 11 02 07" src="https://user-images.githubusercontent.com/26309194/215729413-6118146b-e300-4407-9473-7ae1b50996fe.png">

We already had problems before in production with this, so I'm trying to keep the assignments with `.value` and  leave them as enum types in conditional statements.

Reply:
😮 !! Good to know!